Creating bot and connecting eureka

marl0vv opened the pull request:
- Добавил зависимости для работы с телеграмом
- Подключил бота, добавил токены
- Сделал тестовый вывод "Hello World", чтобы проверить работоспособность
- Пытался подключить Eureka, но словил проблемы с подключением. Походу зависимости для телеграм ботов конфликтуют с зависимостями для Eureka. Не могу разобраться

marl0vv committed:
Set up bot

marl0vv committed:
Implement helloWorld ability

marl0vv committed:
Add function that replies to image

marl0vv committed:
Add eureka client

RavenVSS commented on .gitignore:
```diff
@@ -1,4 +1,5 @@
 HELP.md
+src/main/resources/token.properties
```
nit: Можно в ENV переменные вынести (они же переменные окружения)  

Обычно так и нужно делать для доступов

RavenVSS commented on src/main/java/com/glamik/converter_bot/ConverterBotApplication.java:
```diff
@@ -2,12 +2,28 @@
-		SpringApplication.run(ConverterBotApplication.class, args);
+		ConfigurableApplicationContext ctx = SpringApplication.run(ConverterBotApplication.class, args);
+		try {
+			TelegramBotsApi botsApi = new TelegramBotsApi(DefaultBotSession.class);
```
Вынести это в `configs/BotConfig` и там задай его как бин через метод

marl0vv committed:
Update dependencies

So the developer of the telegrambots library did some strange stuff with fragmentation of his library on many pieces but he didn't provided enough clarity in documentation. That is why at first I used some outdated libraries. Now it is updated to last version and it works with Eureka fine.

marl0vv committed:
Update git ignore

marl0vv committed:
Hide botToken in env variables
